tests/MongoDB.Driver.Tests/Communication/Security/GssapiAuthenticationTests.cs

```diff
@@ -29,64 +29,99 @@ public class GssapiAuthenticationTests
     {
         private static readonly string __collectionName = "test";
 
-        private MongoClientSettings _settings;
+        public string AuthHost => Environment.GetEnvironmentVariable("AUTH_HOST") ?? throw new Exception("AUTH_HOST has not been configured.");
```

Comment:
Properties are not supposed to throw exceptions.

Perhaps a method would be better?

Reply:
Will do it in the next commit